mmdet3d/datasets/s3dis_dataset.py

```diff
@@ -2,14 +2,154 @@
 from os import path as osp
 
 from mmdet3d.core import show_seg_result
+from mmdet3d.core.bbox import DepthInstance3DBoxes
 from mmdet.datasets import DATASETS
 from mmseg.datasets import DATASETS as SEG_DATASETS
+from .custom_3d import Custom3DDataset
 from .custom_3d_seg import Custom3DSegDataset
 from .pipelines import Compose
 
 
 @DATASETS.register_module()
-@SEG_DATASETS.register_module()
+class S3DISDataset(Custom3DDataset):
+    """S3DIS Dataset for Detection Task.
```

Comment:
Need to add an `r` before `"""`, then the link `GSDN <https://arxiv.org/abs/2006.12356>` will be clickable in the generated docs. See this [line](https://github.com/open-mmlab/mmdetection3d/blob/d3213cd3eb5cd64733844d80ca638ee1f0355bb1/mmdet3d/datasets/s3dis_dataset.py#L14).